tests/ludwig/config_validation/test_validate_config_combiner.py

```diff
@@ -42,7 +42,7 @@ def test_config_tabnet(eval_batch_size):
             "validation_field": "label",
         },
     }
-    validate_config(config)
+    validate_config(config, include_auxiliary_validations=False)
```

Comment:
What is the rationale behind setting `include_auxiliary_validations` to `False` for so many of these tests?

Reply:
Good question. `validate_config` takes a dictionary as input, and the unit tests for validate_config are declaring dictionaries in line.

The auxiliary checks in general assume comprehensive configs, which actually seems not always guaranteed.

I've changed this code now to make this guaranteed, and always do auxiliary checks.

Reply:
@justinxzhao out of curiosity, does this make validation slower to any significant degree?

Reply:
Oddly enough the tests seem to run faster, though I've also commented out a few extra `validate_config` tests.

Here are a few runtimes on my dev container:

master: `pytest tests/ludwig/schema/` 28.75s
With changes: `pytest tests/ludwig/schema/` 23.33s

A wash, I'd say.